lib/msal-angular/docs/v2-docs/best-practices.md

```diff
@@ -0,0 +1,121 @@
+# MSAL Angular 2.x Best Practices
+
+Here are some of our optional but recommended practices when using MSAL Angular 2.x.
+
+## Dedicated redirect component
+
+MSAL Angular 2.x provides a dedicated redirect component that can be incorporated into your application. We recommend using this, as `msal.redirect.component.ts` is designed to handle all redirects for your application without your components needing to call `handleRedirectObservable()` manually. 
+
+### Steps for adding the redirect component with hash routing
```

Comment:
Hash routing is the [old way](https://angular.io/guide/router#locationstrategy-and-browser-url-styles) of routing. Maybe place the default browsing first and move the hash routing section down.